programs/ed25519-tests/tests/process_transaction.rs

```diff
@@ -68,7 +70,9 @@ async fn test_failure() {
     assert_matches!(
         client.process_transaction(transaction).await,
         Err(BanksClientError::TransactionError(
-            TransactionError::InvalidAccountIndex
+            TransactionError::InstructionError(0, InstructionError::Custom(3))
         ))
     );
+    // this assert is for documenting the matched error code above
```

Comment:
@jstarry sorry I missed your last comment.

We should still have a test when the feature is disabled.
Is that tested somewhere else that I am missing?

Reply:
Sorry no it's not, I added an extra test for that here: https://github.com/anza-xyz/agave/pull/2441/commits/1084cfab8e21c9ae40245f1982e1e2a6d574a92a

Reply:
perfect